wazo_call_logd/plugins/tenant_migration/http.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from flask import request
+
+from xivo.auth_verifier import required_acl
+from wazo_call_logd.rest_api import AuthResource
+
+
+class CallLogdTenantUpgradeResource(AuthResource):
+
+    def __init__(self, service):
+        self._service = service
+
+    @required_acl('webhookd.tenant-upgrade')
+    def post(self):
+        for item in request.json['users']:
+            self._service.update_participants(**item)
+        for item in request.json['contexts']:
+            self._service.update_contexts(**item)
```

Comment:
We should add a step to remove all non migrated tenant. By deleting the call log and the call_log_id (in CEL table), the cron will automatically regenerate these entries.

Reply:
Well it can also cause some trouble if the resource don't exist anymore. participants information and context can be not fuond... so is maybe not a good idea :/